### PR TITLE
Views wireframe basic

### PR DESCRIPTION
**Pull request by alexansaa**
In this milestone, the following features have been implemented:
- Basic index and show wireframe design implemented on user and post web pages
- Data from the database have been used to render information into the pages
- Basic styles have been used to implement the basic wireframe required
- Different views have been implemented in order to develop the required views designs
- Basic elements like post, user and comment elements have been used through all the project

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,26 @@
...
+  </div>
+  <div class="border mrg">
+    <div class="title mrg">
+      <h2>Bio</h2>
+      <p><%= @user.bio %></p>
+    </div>
+  </div>
+  <div>
+    <% if @posts.present? %>
+      <% @posts.each_with_index do |post, index| %>
```
Comment (Lucas-Masaba) [inline app/views/users/show.html.erb:16]: - [ ] Please use the `most_recent_post` method you created in your User model to render the 3 most recent posts as mentioned in the project requirements.

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1 +1,17 @@
-<h1 name='title'>Here is a list of posts for User Id: <%= @user_id%></h1>
\ No newline at end of file
+<head>
+</head>
+<body>
+  <div>
+    <%= render partial: 'users/user', locals: { user: @user } %>
+  </div>
+  <div>
+    <% @posts.each_with_index do |post, index| %>
```
Comment (Lucas-Masaba) [inline app/views/posts/index.html.erb:8]: - [ ] Please use the `most_recent_comment` method you created in your Post model to render the 5 most recent comments as mentioned in the project requirements.

File: app/views/users/_user.html.erb
Comment (Lucas-Masaba) [inline app/views/users/_user.html.erb:1]: - [ ] Please make sure that clicking on a user should take you to the user show page as mentioned in the project requirements.

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -1 +1,26 @@
...
+        <%= render partial: 'posts/post', locals: { post: post, index: index + 1 } %>
+      <% end %>
+    <% else %>
+      <p>No posts available</p>
+    <% end %>
+  </div>
+  <div class="btn">
+    <button type="button">See all posts!</button>
```
Comment (Lucas-Masaba) [inline app/views/users/show.html.erb]: - [ ] This button should open a user's post index as displayed in the wireframe and project requirements. Currently, it leads to nowhere

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @alexansaa  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Linter checks passing :checkered_flag: 
- Descriptive pull request message :writing_hand: 
- Views look like that in the wireframe :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by abdulhamiid — APPROVED** [review]
# :green_circle:  Approved! :partying_face:
Hi @alexansaa 👋, 
image
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🌟

- ✔️ Descriptions are descriptive
- ✔️ Linters checks are passing
- ✔️ Use preprocessed HTML file with embedded Ruby code.

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**